projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c

```diff
@@ -163,7 +165,21 @@ int main(void)
 						     uint32_t))Xil_DCacheInvalidateRange
 	};
 
-	return iio_app_start(&iio_axi_adc_init_par);
+	status = iio_axi_adc_init(&iio_axi_adc_desc, &iio_axi_adc_init_par);
+	if (status < 0)
+		return status;
+
+	iio_axi_adc_get_dev_descriptor(iio_axi_adc_desc, &dev_desc);
+	struct iio_data_buffer read_buff = {
+		.buff = (void *)ADC_DDR_BASEADDR,
```

Comment:
why not using buffers?

Reply:
This commit is not related to buffer, but indeed that must be changed too.

Reply:
In this manner, you won't have to test future updated projects twice (once for iio_app update and once for iio buffers) 😄